libunwind/src/UnwindCursor.hpp

```diff
@@ -2749,6 +2755,63 @@ int UnwindCursor<A, R>::stepThroughSigReturn(Registers_arm64 &) {
   _isSignalFrame = true;
   return UNW_STEP_SUCCESS;
 }
+#elif defined(_LIBUNWIND_TARGET_HAIKU) && defined(_LIBUNWIND_TARGET_X86_64)
```

Comment:
```
}

#elif ...
#include ...
```

the blank line is preferred before `#elif`, not after

Reply:
Done.